Xamarin.Forms.Platform.Android/ButtonBackgroundTracker.cs

```diff
@@ -77,10 +77,27 @@ public void UpdateDrawable()
 
 				var useDefaultShadow = _button.OnThisPlatform().UseDefaultShadow();
 
-				float shadowRadius = useDefaultShadow ? 2 : _nativeButton.ShadowRadius;
```

Comment:
Not a fan of the nested trinary operator, eh? 

`useDefaultShadow ? 2 : lessThan16 ? 0 : _nativeButton.ShadowRadius;`

Reply:
bit harder to debug, but not bad.